src/main/kotlin/no/skatteetaten/aurora/cantus/service/NexusClient.kt

```diff
@@ -0,0 +1,54 @@
+package no.skatteetaten.aurora.cantus.service
+
+import no.skatteetaten.aurora.cantus.controller.handleError
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import reactor.core.publisher.Mono
+
+@Component
+class NexusClient(
+    webClientBuilder: WebClient.Builder,
+    @Value("\${integrations.nexus.url}") nexusUrl: String
+) {
+
+    private val client = webClientBuilder
```

Comment:
Jeg sitter å funderer om dette heller bør være i configuration?

Reply:
Da sikter du til `ApplicationConfig.kt`, sant? Jeg har ingen umiddelbar formening om det. Hvilke fordeler har man av å plassere det der?

Reply:
Det stemmer. 
Jeg ser at vi ikke har satt noen connect og read timeout her slik vi har pleid. 
Det er vel heller ingen konfigurasjon for TLS her slik som vi har pleid å gjøre. Er ikke dette nødvendig lenger?
Vet du det @jarlehansen ?

Reply:
Jeg har ikke hørt noen endring på dette, er det https mot nexus?
Regner med du har testet dette @TomasEng ?

Reply:
Det ser ut som at det er https basert på application.yaml. 
Jeg kjenner ikke til grunnen for at man genererer nytt sertifikat basert på /var/run/secrets/kubernetes.io/serviceaccount/ca.crt

Vet du dette @stromland ?

Reply:
Jeg har testet, ja, og det fungerer fint.

Reply:
Grunnen er vel de egne https sertifikatene som skatteetaten har generert, men hvis det funker uten så er det ingen problem.